azerothcore/azerothcore-wotlk
# fix(DB/creature): correct spawn position of npc 1568

## Changes Proposed:
Correct spawn position of npc Undertaker Mordo (id: 1568) 
https://classic.wowhead.com/npc=1568/undertaker-mordo

<img width="350" alt="Unbenannt" src="https://user-images.githubusercontent.com/9536270/179397159-c715ca2c-273f-4a6e-aa2e-88a0f29d25a9.PNG">

## Issues Addressed:
- Closes 

## SOURCE:
- VMangos DB

## Tests Performed:
- tested in game

## How to Test the Changes:
1. create new undead character
2. check spawn position of Undertaker Mordo

## Known Issues and TODO List:

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
